{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "cd68a1f1-0648-4503-8548-3dcf826cf5f9",
   "metadata": {},
   "source": [
    "# Unwrap interferograms with snaphu using scaled speckle tracking offsets for coarse phase estimate"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "c54108b5-41a6-4f9d-93b3-545176d130c8",
   "metadata": {},
   "outputs": [],
   "source": [
    "import xarray as xr\n",
    "import rasterio as rio\n",
    "import rioxarray\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "from osgeo import gdal, gdal_array\n",
    "from skimage import data, filters\n",
    "from scipy.interpolate import interpn\n",
    "import os\n",
    "from glob import glob\n",
    "from datetime import datetime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "3062cbe9-7d7c-4e6b-85ce-3ed7ed6bbafd",
   "metadata": {},
   "outputs": [],
   "source": [
    "def prep_inputs(igram_dir, speckle_dir, tbaseline):\n",
    "    '''\n",
    "    - open interferogram, coherence, and speckle tracking product\n",
    "    - resample and pad speckle tracking offsets to match interferogram\n",
    "    - filter speckle tracking product, convert to phase, and scale to temporal baseline\n",
    "    \n",
    "    '''\n",
    "    # veloc_fn = f'{speckle_dir}/velocityRg.h5'\n",
    "    igram_fn = f'{igram_dir}/filt_fine.int'\n",
    "    cor_fn = f'{igram_dir}/filt_fine.cor'\n",
    "\n",
    "    igram_ds = gdal.Open(igram_fn, gdal.GA_ReadOnly)\n",
    "    igram = igram_ds.GetRasterBand(1).ReadAsArray()\n",
    "    cor_ds = gdal.Open(cor_fn, gdal.GA_ReadOnly)\n",
    "    cor = cor_ds.GetRasterBand(1).ReadAsArray()\n",
    "\n",
    "    # ds = xr.open_dataset(veloc_fn)\n",
    "\n",
    "    # non_zero_rows = np.any(igram != 0, axis=1)\n",
    "    # non_zero_cols = np.any(igram != 0, axis=0)\n",
    "    # igram_crop = igram[non_zero_rows][:, non_zero_cols]\n",
    "    # cor_crop = cor[non_zero_rows][:, non_zero_cols]\n",
    "\n",
    "    # veloc = ds.velocity.values\n",
    "    # non_zero_rows = np.any(veloc != 0, axis=1)\n",
    "    # non_zero_cols = np.any(veloc != 0, axis=0)\n",
    "\n",
    "    # veloc_crop = veloc[non_zero_rows][:, non_zero_cols]\n",
    "\n",
    "    # # Find rows and columns with all zeros and remove them in each direction\n",
    "    # row_sum = np.sum(igram, axis=1)\n",
    "    # col_sum = np.sum(igram, axis=0)\n",
    "    # top_padding = 0\n",
    "    # bottom_padding = 0\n",
    "    # left_padding = 0\n",
    "    # right_padding = 0\n",
    "    \n",
    "    # # Determine top padding\n",
    "    # while row_sum[top_padding] == 0:\n",
    "    #     top_padding += 1\n",
    "    \n",
    "    # # Determine bottom padding\n",
    "    # while row_sum[-1 - bottom_padding] == 0:\n",
    "    #     bottom_padding += 1\n",
    "    \n",
    "    # # Determine left padding\n",
    "    # while col_sum[left_padding] == 0:\n",
    "    #     left_padding += 1\n",
    "    \n",
    "    # # Determine right padding\n",
    "    # while col_sum[-1 - right_padding] == 0:\n",
    "    #     right_padding += 1\n",
    "\n",
    "    # interpolate to match interferogram dimensions\n",
    "    # x_coords = np.linspace(0, veloc_crop.shape[1]-1, veloc_crop.shape[1])\n",
    "    # y_coords = np.linspace(0, veloc_crop.shape[0]-1, veloc_crop.shape[0])\n",
    "    \n",
    "    # Create a mesh grid for the interferogram dimensions\n",
    "    # x_coords_new, y_coords_new = np.meshgrid(\n",
    "    #     np.linspace(x_coords.min(), x_coords.max(), igram_crop.shape[1]),\n",
    "    #     np.linspace(y_coords.min(), y_coords.max(), igram_crop.shape[0])\n",
    "    # )\n",
    "    \n",
    "    # # Perform bilinear interpolation using scipy.interpolate.interpn\n",
    "    # veloc = interpn((y_coords, x_coords), veloc_crop, (y_coords_new, x_coords_new), method=\"linear\")\n",
    "\n",
    "    # # filter to remove some noise\n",
    "    # veloc_filtered = filters.butterworth(veloc,\n",
    "    #                                      cutoff_frequency_ratio=0.35, #between 0 and 0.5\n",
    "    #                                      order=2,\n",
    "    #                                      high_pass=False)\n",
    "    \n",
    "    # # convert to phase for simulated 12 day period\n",
    "    # veloc_unwrapped = (veloc_filtered-0.6)*(tbaseline/365)*(12.5663706/0.05546576) # 0.6 to set ref point\n",
    "    # # pad to original interferogram dimensions\n",
    "    # veloc_unwrapped_full = np.pad(veloc_unwrapped, ((top_padding, bottom_padding), (left_padding, right_padding)), mode='constant')\n",
    "\n",
    "    # # crop to aoi\n",
    "    # veloc_unwrapped_aoi = veloc_unwrapped_full[3250:4750, 2600:4100]\n",
    "    igram_aoi = igram[3250:4750, 2600:4100]\n",
    "    cor_aoi = cor[3250:4750, 2600:4100]\n",
    "\n",
    "    # # save cropped speckle tracking displacement\n",
    "    # veloc_unwrapped_aoi = np.nan_to_num(veloc_unwrapped_aoi, nan=0)\n",
    "    # veloc_unwrapped_aoi = veloc_unwrapped_aoi.astype(np.float32)\n",
    "    # output_file = f'{igram_dir}/veloc_unwrapped_aoi' \n",
    "    # driver_format = \"ISCE\"    # Specify the GDAL format for the output (GeoTIFF in this example)\n",
    "    # rows, cols = veloc_unwrapped_aoi.shape    # Get the number of rows and columns from the NumPy array\n",
    "    # data_type = gdal_array.NumericTypeCodeToGDALTypeCode(veloc_unwrapped_aoi.dtype)  # Convert NumPy data type to GDAL data type\n",
    "    # driver = gdal.GetDriverByName(driver_format)\n",
    "    # output_ds = driver.Create(output_file, cols, rows, 1, data_type)\n",
    "    # band = output_ds.GetRasterBand(1)  \n",
    "    # band.WriteArray(veloc_unwrapped_aoi)\n",
    "    # output_ds = None\n",
    "\n",
    "    # save cropped interferogram\n",
    "    igram_aoi = np.nan_to_num(igram_aoi, nan=0)\n",
    "    igram_aoi = igram_aoi.astype(np.complex64)\n",
    "    output_file = f'{igram_dir}/filt_fine_aoi.int' \n",
    "    driver_format = \"ISCE\"    # Specify the GDAL format for the output (GeoTIFF in this example)\n",
    "    rows, cols = igram_aoi.shape    # Get the number of rows and columns from the NumPy array\n",
    "    data_type = gdal_array.NumericTypeCodeToGDALTypeCode(igram_aoi.dtype)  # Convert NumPy data type to GDAL data type\n",
    "    driver = gdal.GetDriverByName(driver_format)\n",
    "    output_ds = driver.Create(output_file, cols, rows, 1, data_type)\n",
    "    band = output_ds.GetRasterBand(1)  \n",
    "    band.WriteArray(igram_aoi)\n",
    "    output_ds = None\n",
    "\n",
    "    # save cropped coherence\n",
    "    cor_aoi = np.nan_to_num(cor_aoi, nan=0)\n",
    "    cor_aoi = cor_aoi.astype(np.float32)\n",
    "    output_file = f'{igram_dir}/filt_fine_aoi.cor' \n",
    "    driver_format = \"ISCE\"    # Specify the GDAL format for the output (GeoTIFF in this example)\n",
    "    rows, cols = cor_aoi.shape    # Get the number of rows and columns from the NumPy array\n",
    "    data_type = gdal_array.NumericTypeCodeToGDALTypeCode(cor_aoi.dtype)  # Convert NumPy data type to GDAL data type\n",
    "    driver = gdal.GetDriverByName(driver_format)\n",
    "    output_ds = driver.Create(output_file, cols, rows, 1, data_type)\n",
    "    band = output_ds.GetRasterBand(1) \n",
    "    band.WriteArray(cor_aoi)\n",
    "    output_ds = None\n",
    "\n",
    "    # save coherence mask \n",
    "    byte_mask = (cor_aoi>0.25).astype(np.uint8)\n",
    "    output_file = f'{igram_dir}/cor0.25_mask' \n",
    "    driver_format = \"ISCE\"    # Specify the GDAL format for the output (GeoTIFF in this example)\n",
    "    rows, cols = byte_mask.shape    # Get the number of rows and columns from the NumPy array\n",
    "    data_type = gdal_array.NumericTypeCodeToGDALTypeCode(byte_mask.dtype)  # Convert NumPy data type to GDAL data type\n",
    "    driver = gdal.GetDriverByName(driver_format)\n",
    "    output_ds = driver.Create(output_file, cols, rows, 1, data_type)\n",
    "    band = output_ds.GetRasterBand(1)  # Get the single band\n",
    "    band.WriteArray(byte_mask)\n",
    "    output_ds = None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "890764c4-6c15-4e5f-b185-a3c08965858e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function to write to MintPy config file\n",
    "def write_config_file(out_file, CONFIG_TXT, mode='a'): \n",
    "    \"\"\"Write text files\"\"\"\n",
    "    if not os.path.isfile(out_file) or mode == 'w':\n",
    "        with open(out_file, \"w\") as fid:\n",
    "            fid.write(CONFIG_TXT)\n",
    "        print('write to file: {}'.format(out_file))\n",
    "    else:\n",
    "        with open(out_file, \"a\") as fid:\n",
    "            fid.write(\"\\n\" + CONFIG_TXT)\n",
    "        print('add the following to file: \\n{}'.format(CONFIG_TXT))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "f8667cab-48dc-4a89-bba6-a180fd582dd1",
   "metadata": {},
   "outputs": [],
   "source": [
    "CONFIG_TXT = f'''# snaphu configuration file\n",
    "#############################################\n",
    "# File input and output and runtime options #\n",
    "#############################################\n",
    "\n",
    "# Input file name\n",
    "INFILE\tfilt_fine_aoi.int\n",
    "\n",
    "# Input file line length \n",
    "LINELENGTH\t1500\n",
    "\n",
    "# Output file name\n",
    "OUTFILE\tfilt_fine_aoi_noest.unw\n",
    "\n",
    "# Correlation file name\n",
    "CORRFILE\tfilt_fine_aoi.cor\n",
    "\n",
    "# Coarse unwrapped-phase estimate file name (see possible file formats\n",
    "# below).  The array should have the same dimensions as the input\n",
    "# wrapped phase array.\n",
    "#ESTIMATEFILE\tveloc_unwrapped_aoi\n",
    "\n",
    "# Input file of signed binary byte (signed char) values.  \n",
    "#BYTEMASKFILE  cor0.25_mask\n",
    "\n",
    "# Text file to which runtime parameters will be logged.  \n",
    "LOGFILE       snaphu.logfile\n",
    "\n",
    "# Statistical-cost mode (TOPO, DEFO, SMOOTH, or NOSTATCOSTS)\n",
    "STATCOSTMODE\tSMOOTH\n",
    "\n",
    "# Algorithm used for initialization of wrapped phase values.  Possible\n",
    "# values are MST and MCF.  \n",
    "INITMETHOD\tMCF\n",
    "\n",
    "################\n",
    "# File formats #\n",
    "################\n",
    "\n",
    "# Input file format\n",
    "INFILEFORMAT\t\tCOMPLEX_DATA\n",
    "\n",
    "# Output file format\n",
    "OUTFILEFORMAT\t\tFLOAT_DATA\n",
    "\n",
    "# Correlation file format\n",
    "CORRFILEFORMAT\t\tFLOAT_DATA\n",
    "\n",
    "# Unwrapped estimate file format\n",
    "ESTFILEFORMAT\t\tFLOAT_DATA\n",
    "\n",
    "###############################\n",
    "# Connected component control #\n",
    "###############################\n",
    "\n",
    "# Grow connected components mask and write to the output file whose\n",
    "# name is specified here as a string.  The mask is a file of unsigned\n",
    "# integer values with the same number of rows and columns as the\n",
    "# unwrapped interferogram.  The type of integer (1 byte vs. 4 byte) is\n",
    "# specified by the CONNCOMPOUTTYPE keyword, with 1-byte integers being\n",
    "# the default.\n",
    "CONNCOMPFILE            conn_comp\n",
    "\n",
    "# Minimum size of a single connected component, as a fraction (double)\n",
    "# of the total number of pixels in tile.\n",
    "MINCONNCOMPFRAC \t0.00001\n",
    "\n",
    "# End of snaphu configuration file'''\n",
    "config_file = 'snaphu.conf.brief'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "354b44b2-07d6-4f1a-8253-50d22b32ef45",
   "metadata": {},
   "outputs": [],
   "source": [
    "unwrapped_xml_txt = '''\n",
    "<imageFile>\n",
    "  <property name=\"WIDTH\">\n",
    "    <value>1500</value>\n",
    "  </property>\n",
    "  <property name=\"LENGTH\">\n",
    "    <value>1500</value>\n",
    "  </property>\n",
    "  <property name=\"NUMBER_BANDS\">\n",
    "    <value>1</value>\n",
    "  </property>\n",
    "  <property name=\"DATA_TYPE\">\n",
    "    <value>FLOAT</value>\n",
    "  </property>\n",
    "  <property name=\"SCHEME\">\n",
    "    <value>BIP</value>\n",
    "  </property>\n",
    "  <property name=\"BYTE_ORDER\">\n",
    "    <value>l</value>\n",
    "  </property>\n",
    "  <property name=\"ACCESS_MODE\">\n",
    "    <value>read</value>\n",
    "  </property>\n",
    "  <property name=\"FILE_NAME\">\n",
    "    <value>filt_fine_aoi_noest.unw</value>\n",
    "  </property>\n",
    "  <component name=\"Coordinate1\">\n",
    "    <factorymodule>isceobj.Image</factorymodule>\n",
    "    <factoryname>createCoordinate</factoryname>\n",
    "    <doc>First coordinate of a 2D image (width).</doc>\n",
    "    <property name=\"name\">\n",
    "      <value>ImageCoordinate_name</value>\n",
    "    </property>\n",
    "    <property name=\"family\">\n",
    "      <value>ImageCoordinate</value>\n",
    "    </property>\n",
    "    <property name=\"size\">\n",
    "      <value>1500</value>\n",
    "    </property>\n",
    "  </component>\n",
    "  <component name=\"Coordinate2\">\n",
    "    <factorymodule>isceobj.Image</factorymodule>\n",
    "    <factoryname>createCoordinate</factoryname>\n",
    "    <property name=\"name\">\n",
    "      <value>ImageCoordinate_name</value>\n",
    "    </property>\n",
    "    <property name=\"family\">\n",
    "      <value>ImageCoordinate</value>\n",
    "    </property>\n",
    "    <property name=\"size\">\n",
    "      <value>1500</value>\n",
    "    </property>\n",
    "  </component>\n",
    "</imageFile>\n",
    "'''\n",
    "unwrapped_xml_file = 'filt_fine_aoi_noest.unw.xml'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "2559ad89-46f6-4560-ad69-22a86d368f56",
   "metadata": {},
   "outputs": [],
   "source": [
    "conn_comp_xml_txt = '''\n",
    "<imageFile>\n",
    "  <property name=\"WIDTH\">\n",
    "    <value>1500</value>\n",
    "  </property>\n",
    "  <property name=\"LENGTH\">\n",
    "    <value>1500</value>\n",
    "  </property>\n",
    "  <property name=\"NUMBER_BANDS\">\n",
    "    <value>1</value>\n",
    "  </property>\n",
    "  <property name=\"DATA_TYPE\">\n",
    "    <value>BYTE</value>\n",
    "  </property>\n",
    "  <property name=\"SCHEME\">\n",
    "    <value>BIP</value>\n",
    "  </property>\n",
    "  <property name=\"BYTE_ORDER\">\n",
    "    <value>l</value>\n",
    "  </property>\n",
    "  <property name=\"ACCESS_MODE\">\n",
    "    <value>read</value>\n",
    "  </property>\n",
    "  <property name=\"FILE_NAME\">\n",
    "    <value>conn_comp</value>\n",
    "  </property>\n",
    "  <component name=\"Coordinate1\">\n",
    "    <factorymodule>isceobj.Image</factorymodule>\n",
    "    <factoryname>createCoordinate</factoryname>\n",
    "    <doc>First coordinate of a 2D image (width).</doc>\n",
    "    <property name=\"name\">\n",
    "      <value>ImageCoordinate_name</value>\n",
    "    </property>\n",
    "    <property name=\"family\">\n",
    "      <value>ImageCoordinate</value>\n",
    "    </property>\n",
    "    <property name=\"size\">\n",
    "      <value>1500</value>\n",
    "    </property>\n",
    "  </component>\n",
    "  <component name=\"Coordinate2\">\n",
    "    <factorymodule>isceobj.Image</factorymodule>\n",
    "    <factoryname>createCoordinate</factoryname>\n",
    "    <property name=\"name\">\n",
    "      <value>ImageCoordinate_name</value>\n",
    "    </property>\n",
    "    <property name=\"family\">\n",
    "      <value>ImageCoordinate</value>\n",
    "    </property>\n",
    "    <property name=\"size\">\n",
    "      <value>1500</value>\n",
    "    </property>\n",
    "  </component>\n",
    "</imageFile>\n",
    "'''\n",
    "conn_comp_xml_file = 'conn_comp.xml'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "99abfa8a-6d1b-44a9-9047-9a1cddec4678",
   "metadata": {},
   "outputs": [],
   "source": [
    "# # crop geometry files\n",
    "# def crop_isce_file(input_path, output_path):\n",
    "\n",
    "#     input_ds = gdal.Open(input_path, gdal.GA_ReadOnly)\n",
    "#     input = input_ds.GetRasterBand(1).ReadAsArray()\n",
    "    \n",
    "#     input_crop = input[3250:4750, 2600:4100]\n",
    "    \n",
    "#     # save cropped interferogram\n",
    "#     input_crop = np.nan_to_num(input_crop, nan=0)\n",
    "#     driver_format = \"ISCE\"    # Specify the GDAL format for the output (GeoTIFF in this example)\n",
    "#     rows, cols = input_crop.shape    # Get the number of rows and columns from the NumPy array\n",
    "#     data_type = gdal_array.NumericTypeCodeToGDALTypeCode(input_crop.dtype)  # Convert NumPy data type to GDAL data type\n",
    "#     driver = gdal.GetDriverByName(driver_format)\n",
    "#     output_ds = driver.Create(output_path, cols, rows, 1, data_type)\n",
    "#     band = output_ds.GetRasterBand(1)  \n",
    "#     band.WriteArray(input_crop)\n",
    "#     output_ds = None\n",
    "\n",
    "# work_dir = '/mnt/Backups/gbrench/repos/fusits/nbs/imja/agu_push/AT12/work'\n",
    "# crop_isce_file(f'{work_dir}/merged/geom_reference/hgt.rdr', f'{work_dir}/merged/geom_reference/hgt_aoi.rdr')\n",
    "# crop_isce_file(f'{work_dir}/merged/geom_reference/incLocal.rdr', f'{work_dir}/merged/geom_reference/incLocal_aoi.rdr')\n",
    "# crop_isce_file(f'{work_dir}/merged/geom_reference/lat.rdr', f'{work_dir}/merged/geom_reference/lat_aoi.rdr')\n",
    "# crop_isce_file(f'{work_dir}/merged/geom_reference/lon.rdr', f'{work_dir}/merged/geom_reference/lon_aoi.rdr')\n",
    "# crop_isce_file(f'{work_dir}/merged/geom_reference/los.rdr', f'{work_dir}/merged/geom_reference/los_aoi.rdr')\n",
    "# crop_isce_file(f'{work_dir}/merged/geom_reference/shadowMask.rdr', f'{work_dir}/merged/geom_reference/shadowMask_aoi.rdr')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "d220ebf2-fd10-49a3-ac6d-dd12c6401610",
   "metadata": {},
   "outputs": [],
   "source": [
    "igram_list = glob('/mnt/Backups/gbrench/repos/fusits/nbs/imja/agu_push/AT12/work/merged/interferograms/*')\n",
    "veloc_dir = '/mnt/Backups/gbrench/repos/fusits/nbs/imja/agu_push/AT12/mintpy_offsets'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "1f35cb9a-2a8f-4ce3-9695-8f0ede5d97dc",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on 20191004_20191016, 1/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191004_20191028, 2/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191004_20191109, 3/45\n",
      "temporal baseline: 36 days\n",
      "^C\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 3, in <module>\n",
      "    import re\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/lib/python3.11/re/__init__.py\", line 124, in <module>\n",
      "    import enum\n",
      "  File \"<frozen importlib._bootstrap>\", line 1176, in _find_and_load\n",
      "  File \"<frozen importlib._bootstrap>\", line 1138, in _find_and_load_unlocked\n",
      "  File \"<frozen importlib._bootstrap>\", line 1078, in _find_spec\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 1504, in find_spec\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 1476, in _get_spec\n",
      "  File \"<frozen importlib._bootstrap_external>\", line -1, in find_spec\n",
      "KeyboardInterrupt\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191016_20191028, 4/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191016_20191109, 5/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191016_20191121, 6/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191028_20191109, 7/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191028_20191121, 8/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191028_20191203, 9/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191109_20191121, 10/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191109_20191215, 11/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191121_20191203, 12/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191121_20191215, 13/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191121_20191227, 14/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191203_20191215, 15/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191203_20191227, 16/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191203_20200108, 17/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191215_20191227, 18/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191215_20200108, 19/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191215_20200120, 20/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191227_20200120, 21/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191227_20200201, 22/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200108_20200120, 23/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200108_20200201, 24/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200108_20200213, 25/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200120_20200201, 26/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200120_20200213, 27/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200120_20200225, 28/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200201_20200225, 29/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200201_20200308, 30/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200213_20200225, 31/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200213_20200308, 32/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200213_20200320, 33/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200225_20200308, 34/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200225_20200320, 35/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200225_20200401, 36/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200308_20200320, 37/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200308_20200401, 38/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200308_20200413, 39/45\n",
      "temporal baseline: 36 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200320_20200401, 40/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200320_20200413, 41/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200401_20200413, 42/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191109_20191203, 43/45\n",
      "temporal baseline: 24 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20191227_20200108, 44/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n",
      "working on 20200201_20200213, 45/45\n",
      "temporal baseline: 12 days\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 125, in cmd_line_parse\n",
      "    raise Exception('ERROR: no input file found!')\n",
      "Exception: ERROR: no input file found!\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "No lookup table (longitude or rangeCoord) found in files.\n",
      "Traceback (most recent call last):\n",
      "  File \"/mnt/Backups/gbrench/sw/miniconda3/envs/insar/bin/geocode.py\", line 8, in <module>\n",
      "    sys.exit(main())\n",
      "             ^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 258, in main\n",
      "    inps = cmd_line_parse(iargs)\n",
      "           ^^^^^^^^^^^^^^^^^^^^^\n",
      "  File \"/mnt/Backups/gbrench/sw/insar_tools/MintPy/src/mintpy/cli/geocode.py\", line 140, in cmd_line_parse\n",
      "    raise FileNotFoundError('No lookup table found! Can not geocode without it.')\n",
      "FileNotFoundError: No lookup table found! Can not geocode without it.\n",
      "--------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "for i, igram_dir in enumerate(igram_list):\n",
    "    date = os.path.basename(igram_dir)\n",
    "    print(f'working on {date}, {i+1}/{len(igram_list)}')\n",
    "\n",
    "    tbaseline =  datetime.strptime(date[9:17], '%Y%m%d')-datetime.strptime(date[0:8], '%Y%m%d')\n",
    "    print(f'temporal baseline: {tbaseline.days} days')\n",
    "    \n",
    "    os.chdir(igram_dir)\n",
    "    #if not os.path.exists(f'{igram_dir}/filt_fine_aoi_withest.unw.xml'):\n",
    "    #print('prepping inputs')\n",
    "    #prep_inputs(igram_dir, veloc_dir, tbaseline.days)\n",
    "    #write_config_file(config_file, CONFIG_TXT, mode='w')\n",
    "    #print('************unwrapping************')\n",
    "    #!snaphu -f snaphu.conf.brief\n",
    "    #write_config_file(unwrapped_xml_file, unwrapped_xml_txt, mode='w')\n",
    "    #write_config_file(conn_comp_xml_file, conn_comp_xml_txt, mode='w')\n",
    "    !rm *.rsc\n",
    "    # !geocode.py fine_filt_aoi_noest.unw --lat-file ../../geom_reference/lat_crop.rdr --lon-file ../../geom_reference/lon_crop.rdr\n",
    "    # !geocode.py fine_filt_aoi.cor --lat-file ../../geom_reference/lat_crop.rdr --lon-file ../../geom_reference/lon_crop.rdr\n",
    "    # !geocode.py conn_comp --lat-file ../../geom_reference/lat_crop.rdr --lon-file ../../geom_reference/lon_crop.rdr\n",
    "    # !geocode.py filt_fine_aoi.int --lat-file ../../geom_reference/lat_crop.rdr --lon-file ../../geom_reference/lon_crop.rdr\n",
    "    #else:\n",
    "       # print('unwrapped igram exists, skipping')\n",
    "    print('--------------------------------------------------')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
